packages/mobile/src/import/ImportWallet.tsx

```diff
@@ -134,7 +135,20 @@ export class ImportWallet extends React.Component<Props, State> {
   }
 
   setBackupPhrase = (input: string) => {
+    // Hide the alert banner if one is displayed.
     this.props.hideAlert()
+
+    const updatedPhrase = formatBackupPhraseOnEdit(input)
+
+    const currentWordCount = countMnemonicWords(this.state.backupPhrase)
+    const updatedWordCount = countMnemonicWords(updatedPhrase)
+    if (updatedWordCount !== currentWordCount) {
+      ValoraAnalytics.track(OnboardingEvents.wallet_import_phrase_updated, {
+        wordCount: updatedWordCount,
+        wordCountChange: updatedWordCount - currentWordCount,
+      })
+    }
+
     this.setState({
       backupPhrase: formatBackupPhraseOnEdit(input),
```

Comment:
nit: We can simply use `updatedPhrase` here instead of calling `formatBackupPhraseOnEdit` again

Reply:
Resolved in 87f690d